packages/kujua-sms/views/lib/smsparser.js

```diff
@@ -201,6 +202,8 @@ exports.parse = function (def, doc) {
         return {};
     }
 
+    if(doc.message) doc.message = standardiseDigits(doc.message);
```

Comment:
What do you suggest?  Would it be acceptable to copy `doc.message` to e.g. `doc.original_message`, so it can always be recovered later from the DB if anyone's interested?  Or if we want to display the original, but process the standardised message, then the change will be a bit more invasive...